tests/test_matrix_utils_core.py

```
@@ -133,7 +134,7 @@ def test_get_interaction_settings_none_config():
 def test_get_interaction_settings_invalid_top_level_config_type() -> None:
     """Non-dict top-level config values should disable interactions."""
     with patch("mmrelay.matrix_utils.logger") as mock_logger:
-        result = get_interaction_settings(True)  # type: ignore[arg-type]
+        result = get_interaction_settings(cast(dict, True))
```

Ruff (0.15.12)

[warning] Boolean positional value in function call

(FBT003)